content/blog/dotnet-preview/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Infrastructure as Code with .NET and Pulumi"
+date: 2019-11-13T11:23:04-06:00
+draft: false
+meta_desc: "Pulumi supports .NET Core languages for Infrastructure as Code"
```

Comment:
Maybe make it more... active? Like, `Pulumi launches the support`....

Reply:
accepted